### DISCUSSION TITLE
Welcome to Open Research Calendar Discussions!

### DISCUSSION THREAD

**Discussion by b-kennedy0** [openresearchcalendar/Open-Research-Calendar#43] [General]
## 👋 Welcome! ✏️
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
- Ask questions without needing to add an Issue
- Share ideas and improvement suggestions
- Engage with other community members and their suggestions

**Comment by SK-OOR** [comment]
Hi, I clicked the iCal link from your website to add the OR calendar to my work Outlook (I produce a monthly Open Research Digest at the University of Manchester, so thought it would be a good idea to stay up to date). But, now it is automatically overlaid over my own work calendar, and even if I delete the events they automatically come back - and I assume new events will keep being added in future. 

I really would like to know how to disconnect from this auto-updating calendar, but Reddit and ChatGPT have been no help so far. They either suggest I have subscribed to the calendar and I should see it in my 'Other calendars' list to turn off (which it isn't), or that it isn't auto-updating and you can simply delete the events and job done (which isn't the case either!)

Any help appreciated

**Reply by b-kennedy0** [reply]
Hi @SK-OOR, I've been having a look into this for you. It looks like when you were adding the ICS file, it should have given you the option to choose which calendar to import to. It looks as though Microsoft do not have a feature to 'Undo' an import. Instead, the suggestion is as follows:

[You may need to switch to Outlook Classic to do this] - Navigate to list view on the calendar. In View Settings, you can add/remove columns. Choose to add the organiser and modified date fields to the view. The organiser field may show your own name, which is not helpful here. However, the modified date should show a series of events all added at the same time. In which case, sort by modified date and delete everything that came in at the same time.

I've tested this (on a temporary calendar that I created), and once I removed all the events, no matter how many times I clicked to sync or refresh the folder, the events did not return. 

Let me know how you get on with this!
